pytket/extensions/qiskit/backends/ibmq_emulator.py

```diff
@@ -83,19 +80,16 @@ def __init__(
         super().__init__()
         self._ibmq = IBMQBackend(
             backend_name=backend_name,
-            hub=hub,
-            group=group,
-            project=project,
-            account_provider=account_provider,
+            instance=instance,
+            provider=provider,
             token=token,
         )
 
         self._service = QiskitRuntimeService(channel="ibm_quantum", token=token)
         self._session = Session(service=self._service, backend="ibmq_qasm_simulator")
 
         # Get noise model:
-        aer_sim = AerSimulator.from_backend(self._ibmq._backend)
-        self._noise_model = NoiseModel.from_backend(aer_sim)
+        self._noise_model = NoiseModel.from_backend(self._ibmq._backend)
```

Comment:
Are we all satisfied that this is an okay way to get the `NoiseModel` for the `IBMQEmulatorBackend` instead of going via Aer?